.env.ganache

```diff
@@ -9,7 +9,7 @@ PROVIDER_HOST=127.0.0.1
 PROVIDER_PORT=8545
 
 # Schelling Coin Address (if want to re-use already deployed instance)
-SCHELLING_COIN_ADDRESS=0xB007d2B950467c397846e7797675B76128f8B2C4
+SCHELLING_COIN_ADDRESS=
```

Comment:
? Unrelated change?

Reply:
Whenever we deploy on ganache(for local-testing), we don't have existing ` SCHELLING_COIN_ADDRESS` & won't be reusing it. So Ganache deployment should be from scratch but if someone wants redeployment, he can easily modify the `.env.ganache` according to their needs.